packages/core/echo/echo-db/src/testing/echo-test-builder.ts

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { Level } from 'level';
+
+import { type Context, Resource } from '@dxos/context';
+import { type LevelDB } from '@dxos/echo-pipeline';
+import { PublicKey } from '@dxos/keys';
+import { type Storage, StorageType, createStorage } from '@dxos/random-access-storage';
+
+import { EchoClient } from '../client';
+import { EchoHost } from '../host';
+
+export class EchoTestBuilder extends Resource {
+  private readonly _peers: EchoTestPeer[] = [];
+
+  protected override async _close(ctx: Context): Promise<void> {
+    await Promise.all(this._peers.map((peer) => peer.close(ctx)));
+  }
+
+  async createPeer(): Promise<EchoTestPeer> {
+    const peer = new EchoTestPeer();
+    this._peers.push(peer);
+    await peer.open();
+    return peer;
+  }
+}
+
+export class EchoTestPeer extends Resource {
+  private readonly _kv: LevelDB;
```

Comment:
```suggestion
  private readonly _keyValueStore: LevelDB;
```

Reply:
KV is pretty standard shorthand for a key-value store (same as DB for database). How do you feel about leaving it in?